src/vireshJJun20/Assignment22/PalindromeInteger.java

```diff
@@ -0,0 +1,37 @@
+/*
+Program 1: 
+Verify given number is palindrome or not. 
+Input: number = 45654
+Output : Given number 45654 is palindrome.
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.
+*/
+
+
+package vireshJJun20.Assignment22;
+
+import java.util.Scanner;
+
+public class PalindromeInteger {
+
+	void findPalindromeInt(int input){ //123
+		int revNum=0, mod=0, origNum= input ;
+		while(input!=0){
+			mod= input%10; // 3 >> 2 >> 1
+			revNum= revNum*10 + mod; // 3 >> 32 >> 321 
```

Comment:
instead of mod variable we can directly use like below ,

revNum= (revNum*10) + (input%10)

Logic and code looks perfect otherwise

Reply:
Yes, I got your point.. Thanks.

![image](https://user-images.githubusercontent.com/66830247/86261113-c674d980-bbdb-11ea-9f53-e259af3f7d3b.png)


> it